composer.json

```diff
@@ -21,17 +21,15 @@
             "email": "zoohie@gmail.com"
         }
     ],
-    "minimum-stability": "dev",
-    "prefer-stable": true,
     "require": {
         "php": ">=8.1",
         "amphp/amp": "^3",
         "amphp/byte-stream": "^2",
-        "amphp/http-client": "^5",
+        "amphp/http-client": "^v5.0.0-beta.11",
```

Comment:
Should this be `^v5.0.0-beta.11` or `v5.0.0-beta.11`?  I don't think there's a difference but I'm not positive.